lib/package-json-reader.js

```diff
@@ -19,26 +19,16 @@ export default reader
  * @returns {{string: string|undefined}}
  */
 function read(jsonPath) {
-  return find(path.dirname(jsonPath))
-}
-
-/**
- * @param {string} dir
- * @returns {{string: string|undefined}}
- */
-function find(dir) {
   try {
     const string = fs.readFileSync(
-      path.toNamespacedPath(path.join(dir, 'package.json')),
```

Comment:
So this solves your problem? It looks so small that that seems somewhat unexpected!